examples/workflow/3_perform_fit.ipynb

```diff
@@ -0,0 +1,375 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Download this notebook :download:`here <3_perform_fit.ipynb>`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 3: Perform fit"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "As explained in the :doc:`previous step <2_generate_data>`, an\n",
+    ":class:`~.Intensity` object behaves just like a mathematical function that\n",
+    "takes a :class:`~.DataSet` as an argument and returns a list of intensities\n",
+    "(real numbers). We now want to optimize the parameters of the intensity model\n",
+    "so that it matches the distribution of our data sample. This is what we call\n",
+    "'performing a fit'."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "    We first load the relevant data from the previous steps."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pycompwa.ui as pwa"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "particle_list = pwa.read_particles('model.xml')\n",
+    "kinematics = pwa.create_helicity_kinematics('model.xml', particle_list)\n",
+    "kinematics.create_all_subsystems()\n",
+    "data_sample = pwa.read_root_data(input_file='generated_data.root')\n",
+    "phsp_sample = pwa.read_root_data(input_file='generated_phsp.root')\n",
+    "intensity_builder = pwa.IntensityBuilderXML(\n",
+    "    'model.xml', particle_list, kinematics, phsp_sample)\n",
+    "intensity = intensity_builder.create_intensity()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.1 Define estimator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To perform a fit, you need to define an **estimator**. An estimator is a measure for the discrepancy between the intensity model and the data distribution to which you fit. To create an estimator we therefore generally need:"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "- an :class:`pycompwa.ui.Intensity` instance\n",
+    "- a :class:`pycompwa.ui.DataSet` to which we fit that intensity\n",
+    "- optionally: a phase space :class:`~.DataSet` if you\n",
+    "  want to normalize the :class:`~.Intensity`"
```

Comment:
Remove that, because in our current design the normalization is already included in the Intensity.

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/c8a49675e989f3ec982584cbd916812bc948dccf